changelogs/fragments/55903_kubevirt.yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
```

Comment:
Shouldn't that be:
```suggestion
minor_changes:
```
? I don't see how this is a bugfix.

Reply:
This parameter was supposed to be `interface_name`. I am waiting for Kubevirt maintainers to comment and will update accordingly.